index.js

```diff
@@ -378,10 +388,64 @@ class McEncryptionSchemePolyfill {
       capabilities.keySystemAccess =
           new EmeEncryptionSchemePolyfillMediaKeySystemAccess(
               capabilities.keySystemAccess, supportedScheme);
+    } else if (requestedConfiguration.keySystemConfiguration) {
+      const mediaKeySystemConfig =
+          McEncryptionSchemePolyfill.convertToMediaKeySystemConfig_(
+              requestedConfiguration);
+      capabilities.keySystemAccess =
+          await navigator.requestMediaKeySystemAccess(
+              requestedConfiguration.keySystemConfiguration.keySystem,
+              [mediaKeySystemConfig]);
     }
 
     return capabilities;
   }
+
+  /**
+   * Convert the MediaDecodingConfiguration object to a
+   * MediaKeySystemConfiguration object.
+   * @param {!MediaDecodingConfiguration} decodingConfig
+   * @return {!MediaKeySystemConfiguration}
+   */
+  static convertToMediaKeySystemConfig_(decodingConfig) {
+    const mediaCapkeySystemConfig = decodingConfig.keySystemConfiguration;
+    const audioCapabilities = [];
+    const videoCapabilities = [];
+
+    if (mediaCapkeySystemConfig.audio) {
+      const capability = {
+        robustness: mediaCapkeySystemConfig.audio.robustness || '',
+        contentType: decodingConfig.audio.contentType,
+      };
+      audioCapabilities.push(capability);
+    }
+
+    if (mediaCapkeySystemConfig.video) {
+      const capability = {
+        robustness: mediaCapkeySystemConfig.video.robustness || '',
+        contentType: decodingConfig.video.contentType,
+      };
+      videoCapabilities.push(capability);
+    }
+
+    /** @type {!MediaKeySystemConfiguration} */
+    const mediaKeySystemConfig = {
+      initDataTypes: [mediaCapkeySystemConfig.initDataType],
```

Comment:
This is allowed to be an empty list in EME, and it's allowed to be undefined in MCap.

In native MCap, it defaults to an empty string if it's undefined, but neither undefined nor the empty string are allowed as array members in EME.